src/style.css

```diff
@@ -0,0 +1,32 @@
+* {
+  margin: 0;
```

Comment:
if you want to reset margins and paddings do it on elements you need

Reply:
But i need do it in every element in file. So it will be appropriate such duplicate ?